src/azul/__init__.py

```diff
@@ -825,6 +825,13 @@ def __attrs_post_init__(self):
             if self.internal:
                 assert self.is_integration_test_catalog is True, self
 
+            atlases = {
+                Plugin.atlas_for_type(type_name, plugin.name)
+                for type_name, plugin in self.plugins.items()
+            }
+            require(len(atlases) == 1,
+                    'Conflicting plugin atlases', atlases)
```

Comment:
Unfortunately there appears to be an inconsistency in how we use the term "atlas". In the ticket, a dichotomy was established between AnVIL and HCA, but in our configuration files, we also use "lumgmap" as an atlas. I couldn't use `self.atlas` here because for lungmap catalogs, `self.atlas` is "lungmap", not "hca", causing the requirement to fail. It seems to me that "atlas" is distinct from the concept of "metadata schema" and that we may have been improperly using the terms interchangeably. I had meant to bring this up in PL before requesting review but it appears I forgot.

Reply:
Decided at PL to compare `BUNDLE` generic parameter between the plugins instead of atlas name